src/main/kotlin/apply/domain/mission/ProgrammingLanguage.kt

```diff
@@ -0,0 +1,8 @@
+package apply.domain.mission
+
+enum class ProgrammingLanguage(val language: String) {
+    NONE(""),
+    JAVA("Java"),
+    JAVASCRIPT("Javascript"),
```

Comment:
a: JS는 이렇게 작성하지 않나요? ㅋㅋ 

```suggestion
    JAVASCRIPT("JavaScript"),
```